### PR TITLE
Items page table improvement

### PR DESCRIPTION
**Pull request by mikeCyb**

### CODE REVIEW

File: apps/expo/app.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
         "@rnmapbox/maps",
         {
           "RNMapboxMapsImpl": "mapbox",
-          "RNMapboxMapsDownloadToken": "MAPBOX_DOWNLOADS_TOKEN_FROM_ENV"
+          "RNMapboxMapsDownloadToken": "sk.eyJ1IjoiYW5kcmV3Ymllcm1hbiIsImEiOiJjbGZkZGV5cGExd2Q1NDJsN3JoM2syc3RoIn0.t5eEVBt8Kz1CJ37NPu0fmw"
```
Comment (andrew-bierman) [inline apps/expo/app.json:31]: revert

**Comment by andrew-bierman** [comment]
Does pagination still work on this? If so it looks good to merge 

See comment on token

**Review by JewelTee — COMMENTED** [review]
@mikeCyb  Pagination Arrows and Page Numbers are missing on the Items page for both web and mobile apps. Please check the images below. Thanks! 
Screenshot 2024-07-22 141221
Screenshot_1721630118

**Comment by EngrAJ8888** [comment]
Hi @mikeCyb, I've encountered the same errors mentioned above.

File: apps/expo/app.json (hunk 1 above)
Comment (mikeCyb) [reply apps/expo/app.json]: Sorry, it's fixed now.

**Comment by mikeCyb** [comment]
Hey @JewelTee and @EngrAJ8888,

Andrew asked me to use the same table as we have on the pack details page and clean up the UI a bit. That's why some of its features have been removed.

**Comment by JewelTee** [comment]
@mikeCyb got it! Can we update this and add the Pagination Arrows and Page Numbers? Thank You

**Comment by mikeCyb** [comment]
@JewelTee Thanks for the feedback. It's now updated, and I also fixed the issue with the loading screen box size shrinking.

**Comment by andrew-bierman** [comment]
Merging in but please take a look @EngrAJ8888 and @JewelTee 

Also resizing might cause table to flow outside of container FYI for a later update. not super worried about it now but would be nice to tidy up

**Review by andrew-bierman — APPROVED** [review]
nice job

**Pull request merged**